src/Sylius/Bundle/PayumBundle/Resources/config/services.xml

```diff
@@ -20,11 +20,17 @@
         <import resource="services/action.xml" />
         <import resource="services/controller.xml" />
         <import resource="services/extension.xml" />
+        <import resource="services/form.xml" />
     </imports>
 
     <services>
         <service id="sylius.payum.http_client" class="Payum\Core\HttpClientInterface">
             <factory class="Payum\Core\Bridge\Guzzle\HttpClientFactory" method="create" />
         </service>
+
+        <service id="sylius.registry.payum_gateway_config_type" class="Sylius\Component\Registry\ServiceRegistry">
```

Comment:
Of course we can - I don't know why I wanted to create a custom registry for these form types (that change will also make `GatewayConfigurationTypeInterface` unneeded) 🐃